src/runtime/components/forms/Input.vue

```diff
@@ -193,6 +193,10 @@ export default defineComponent({
         value = looseToNumber(value)
       }
 
+      if (modelModifiers.value.nullable && !value) {
+        value = null
+      }
```

Comment:
Is the custom modifier here really necessary? Updating prop types is enough to fix the issue.

Reply:
If we move forward with it, we need to implement it on all input components. I'd suggest renaming it to `nullify` too.

Reply:
The modifier is the primary reason for this PR. When users type something in an optional input field but then later remove it, the value will be an empty string instead of `null` or `undefined`. Sometimes I want the behavior to be that the value is `null`

Reply:
Let's stick with the type changes allowing null for all inputs in this PR. We can address the nullable modifier in a separate discussion, but I don't see enough use cases to include it in the library. Users can handle coercion themselves if needed.

Reply:
Hmm, the modifier what kinda the whole point of this PR.
Most of the time when I work with forms, I have some sort of state object like so:
```ts
const state = ref({
  name: '',
  email: '',
  company: null,
  title: null,
});
```
Currently, doing coercion in a `useFetch` function needs destructuring:
```ts
const { data } = await useFetch('/api/some/endpoint', {
  method: 'POST',
  body: {
    name: state.value.name,
    email: state.value.email,
    company: state.value.company ?? null,
    title: state.value.title ?? null,
  },
});
```
Now if you have some kind of dashboard with lots and lots of forms (my case), this becomes quite tedious.
Adding this modifier will remove so much unnecessary spreads, destructuring, and coercions.